src/open_api.js

```diff
@@ -0,0 +1,7 @@
+import { createRequire } from 'module'
+
+// TODO: remove once Node.js supports JSON imports with pure ES modules without
+// any experimental flags
+const require = createRequire(import.meta.url)
+
+export const openApiSpec = require('@netlify/open-api')
```

Comment:
`@netlify/open-api` [exports a JSON file](https://github.com/netlify/open-api/blob/2245d381e36931984eae2232466d6e56d23fde57/package.json#L46). However, importing JSON files with pure ES modules is not currently supported without experimental flags.

We could make `@netlify/open-api` export a JavaScript file instead.

Instead, this PR opts for a faster approach: using `module.createRequire()` until JSON imports become possible.

Please also note that `import.meta.resolve()` is not possible since `openApiSpec` must be available synchronously.